packages/ui/src/containers/result-data/experimentation-guide.tsx

```diff
@@ -151,11 +151,14 @@ export const ExperimentationGuide = (props: ResultDataProps) => {
         <Suggestions
           values={nextValues}
           headers={headers}
-          onCopyToDataPoints={index =>
-            dispatchExperiment({
-              type: 'copySuggestedToDataPoints',
-              payload: [index],
-            })
+          onCopyToDataPoints={
+            !isInitializing
```

Comment:
Maybe add an optional (allowIndividualTransfer) prop to the ExperimentationGuide component so that the feature can be controlled by users of the library

Reply:
I actually started writing it but then didn't, because I'm lazy 🙂 I'll add it.